x/evm/types/params_test.go

```diff
@@ -47,9 +48,14 @@ func TestParamsValidate(t *testing.T) {
 		},
 		{
 			"invalid chain config",
-			NewParams("ara", true, true, ChainConfig{}, 2929, 1884, 1344),
+			NewParams("ara", true, true, ChainConfig{}, sdk.NewDecWithPrec(50, 2), 2929, 1884, 1344),
 			false,
 		},
+		{
+			"invalid min gas denominator",
+			NewParams("ara", true, true, ChainConfig{}, sdk.NewDec(2), 2929, 1884, 1344),
```

Comment:
Shouldn't you use a valid `chainConfig` here?

Reply:
This test is useless. Just removed it, good catch